catalog/app/containers/Bucket/Search.js

```diff
@@ -290,15 +312,19 @@ function Search({ bucket, query, page, mode, retry }) {
         >
           <QueryInput query={query || ''} bucket={bucket} onChange={handleQueryChange} />
         </M.Box>
-        <M.Box flexGrow={1} display={{ xs: 'none', sm: 'block' }} />
+        <M.Box
+          flexGrow={1}
+          display={{ xs: 'none', sm: 'block' }}
+          className={classes.stats}
+        >
+          {query && <SearchStats data={data} />}
```

Comment:
i think we should cast `query` to boolean here to be safe
```suggestion
          {!!query && <SearchStats data={data} />}
```